handyrl/train.py

```diff
@@ -349,8 +349,8 @@ def shutdown(self):
 
     def train(self):
         if self.optimizer is None:  # non-parametric model
-            print()
-            return
+            time.sleep(0.1)
```

Comment:
Why is this needed?

Reply:
This is because without time.sleep(), train() will be called incessantly.
It may be better to prevent calling this function.

Reply:
The problem is that epoch will not proceed until train() is called.